src/extension/app/store/site.js

```diff
@@ -168,6 +168,12 @@ export class SiteStore {
    */
   authTokenExpiry;
 
+  /**
+   * Has the store been initialized?
+   * @type {boolean}
+   */
+  ready = false;
```

Comment:
Could do
`@observable accessor ready = false;`

Reply:
@dylandepass I wasn't able to make this work: I think I might have to extend `SiteStore` from some class in mobx but couldn't figure out which one... so I left the `requestUpdate()` call in `aem-sidekick.js` for now. If you help me find the problem with making `ready` observable, I'll gladly change it before merging the PR.